Core/PrimaryGenerator.cpp

```diff
@@ -86,7 +87,7 @@ G4PrimaryVertex* PrimaryGenerator::GenerateThreeGammaVertex(
 }
 
 G4PrimaryVertex* PrimaryGenerator::GenerateTwoGammaVertex(
-  const G4ThreeVector vtxPosition, const G4double T0,  const G4double lifetime2g) 
+  const G4ThreeVector vtxPosition, const G4double T0,  const G4double lifetime2g)
```

Comment:
If you could fix: it should be `(const G4ThreeVector& vtxPosition, const G4double T0,  const G4double lifetime2g)`

Reply:
Ok, also added & to other methods, where necessary